schedule-replicas-by-topology-labels.md

```diff
@@ -0,0 +1,99 @@
+
+## Configure `labels` based on the cluster topology
+
+### Set the `label` configuration of TiKV
+
+TiKV supports binding some attributes through command-line flags or configuration files in the form of key-value pairs. The attribute is called a `label`. TiKV reports its labels to PD so we can mark the TiKV location.
+
+Assume that the topology has three structures: zone > rack > host, use these labels to set the TiKV location.
+
+Command-line flags:
+
+{{< copyable "" >}}
+
+```
+tikv-server --labels zone=<zone>,rack=<rack>,host=<host>
+```
+
+Configuration files:
+
+{{< copyable "" >}}
+
+```toml
+[server]
+labels = "zone=<zone>,rack=<rack>,host=<host>"
+```
+
+### Set the `location-labels` configuration of PD
+
+According to the previous description, labels can be any key-value pairs used to describe TiKV attributes, but PD can not distinguish the location-marking labels and the structure among them. Therefore, you need set some configuration to understand the TiKV node topology.
+
+Set the PD configuration `location-labels` through the PD configuration files.
+
+{{< copyable "" >}}
+
+```toml
+[replication]
+location-labels = ["zone", "rack", "host"]
+```
+
+If PD cluster is already initialized, you need to use the pd-ctl tool to make online changes:
+
+{{< copyable "shell-regular" >}}
+
+```bash
+pd-ctl config set location-labels zone,rack,host
+```
+
+The `location-labels` configuration is an array of strings, and each item corresponds to the key of TiKV `labels`. The sequence of each key represents the layer relationship among different labels.
+
+> **Note:**
+>
+> You must configure `location-labels` for PD and `labels` for TiKV at the same time for `labels` to take effect. Otherwise, PD does not schedule according to the topology.
+
+### Use TiDB Ansible to deploy a cluster
+
+You can directly set location related configuration in the `inventory.ini` file when using TiDB Ansible to deploy a cluster. `tidb-ansible` generates the corresponding TiKV and PD configuration files during deployment.
+
+The following example defines a double-level topology of `zone/host`. The TiKV of the cluster is distributed in three zones, with two hosts in each zone. z1 is deployed with two TiKV instances per host, and z2 and z3 are deployed with 1 instance per host.
+
+```
+[tikv_servers]
+# z1
+tikv-1 labels="zone=z1,host=h1"
+tikv-2 labels="zone=z1,host=h1"
+tikv-3 labels="zone=z1,host=h2"
+tikv-4 labels="zone=z1,host=h2"
+# z2
+tikv-5 labels="zone=z2,host=h1"
+tikv-6 labels="zone=z2,host=h2"
+# z3
+tikv-7 labels="zone=z3,host=h1"
+tikv-8 labels="zone=z3,host=h2"
+[pd_servers:vars]
+location_labels = ["zone", "rack"]
```

Comment:
@ireneontheway There is quite a bit of outdated information in your translated `schedule-replicas-by-topology-labels.md` file. The PR https://github.com/pingcap/docs-cn/pull/3174 you translate was merged 3 months ago and changes are made to this file during the 3 months. Would you check [the change history](https://github.com/pingcap/docs-cn/commits/master/schedule-replicas-by-topology-labels.md) and update your PR? Please make sure you are aligning the latest content instead of the content composed 3 months ago? **Please base your translation on the latest version (upstream).**
```suggestion
location_labels = ["zone", "host"]
```

Reply:
Mainly done in https://github.com/pingcap/docs/pull/3825/commits/eb1120e75f1b2edbb57f289125cbebea1cee0dc7